### PR TITLE
Typescript

### LINKED ISSUE #2: TypeScript compatibility
I'm using this component in a TypeScript-based app. When running `npx svelte-check`, I get the following TypeScript compiler error:
```
src/App.svelte:26:35
Error: Cannot find module 'svelte-collapsible' or its corresponding type declarations. (ts)
  import { CollapsibleCard } from "svelte-collapsible";
```
I believe this could be fixed by exporting types. My initial search regarding what changes would be required came up with https://github.com/sveltejs/component-template/issues/29 and this linked typed template: https://github.com/mattjennings/svelte-typescript-component-template (which ensures it can be consumed by both typescript and plain javascript apps) - would you be willing to make the corresponding changes to your component package?

### PR DESCRIPTION
**Pull request by st--**
Resolves #2

### CODE REVIEW

**Comment by st--** [comment]
@rsdavis I'm not sure if this is already sufficient to support typescript... I'm not very familiar with npm and not sure how I'd test this locally. Let me know if you've got any thoughts, or if we should just bump the version number & try it out ?

**Comment by st--** [comment]
I've merged #12 into this to (hopefully) make it easier to merge them all. It should not have any conflicts with #11.

**Review by rsdavis — APPROVED** [review]
lgtm

File: package.json [hunk 1]
```diff
@@ -2,10 +2,11 @@
   "name": "svelte-collapsible",
   "description": "A collection of high-level Svelte components designed for expanding and collapsing content.",
   "keywords": ["svelte", "component", "accordion", "collapse", "expand"],
-  "version": "0.2.1",
+  "version": "0.2.2",
   "svelte": "src/index.js",
   "module": "dist/index.mjs",
   "main": "dist/index.js",
+  "types": "dist/index.d.ts",
```
Comment (st--) [inline package.json]: actually, would this have to be
```suggestion
  "types": "src/index.d.ts",
```
?

Comment (rsdavis) [reply package.json]: Youre right, this will need to point to `src`. If you build the package `npm run build` you can see that types are not included

**Pull request merged**